mi/dataset/parser/utilities.py

```diff
@@ -121,6 +121,25 @@ def timestamp_yyyymmddhhmmss_to_ntp_time(timestamp_str):
 
     return (timestamp - datetime(1900, 1, 1)).total_seconds()
 
+def timestamp_yyyy_mm_dd_hh_mm_ss_to_ntp_time(timestamp_str):
```

Comment:
refactor `string_to_ntp` or `text_to_ntp` and have the user provide the appropriate format string.